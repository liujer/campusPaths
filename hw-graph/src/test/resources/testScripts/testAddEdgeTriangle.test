# Tests adding of edges by using a three node graph and
# connecting them in a triangle formation
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
AddEdge A one two e12
AddEdge A two three e23
AddEdge A three one e31
ListNodes A
ListChildren A one
ListChildren A two
ListChildren A three



