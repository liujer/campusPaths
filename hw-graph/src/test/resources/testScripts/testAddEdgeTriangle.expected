# Tests adding of edges by using a three node graph and
# connecting them in a triangle formation
Created graph A
Added node one to A
Added node two to A
Added node three to A
Added edge e12 to A
Added edge e23 to A
Added edge e31 to A
ListNodes of A output: one three two
ListChildren of one in A output: two
ListChildren of two in A output: three
ListChildren of three in A output: one





